packages/mdc-snackbar/package.json

```diff
@@ -16,7 +16,12 @@
   "dependencies": {
     "@material/animation": "^0.41.0",
     "@material/base": "^0.41.0",
+    "@material/button": "^0.41.0",
+    "@material/icon-button": "^0.41.0",
+    "@material/dom": "^0.41.0",
```

Comment:
I take it you're expecting to use this in the JS portion?

Reply:
Yep, I just forgot to remove it from this PR. Fixed!